### PR TITLE
DOC-3170 :: Feature :: DevOps Scripts standard Harmony Reference

### PR DESCRIPTION
**Pull request by Alex-DA**
**Merge / Pull request information:**

* Asana task link: https://app.asana.com/0/1200574432121296/1200996275923170/f

### CODE REVIEW

**Review by joselufo — APPROVED** [review]

**Review by lucianosantana — APPROVED** [review]

File: project-example/bin/helper/common.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+echo "Common Scripts..."
+
+#!/bin/bash
```
Comment (jgimenez) [inline project-example/bin/helper/common.sh:4]: duplicate

File: project-example/bin/helper/common.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+echo "Common Scripts..."
```
Comment (jgimenez) [inline project-example/bin/helper/common.sh:2]: why would you want to print this?

File: project-example/bin/helper/common.sh [hunk 3]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```
Comment (jgimenez) [inline project-example/bin/helper/common.sh:1]: use `-e` in all scripts,  to ensure commands finish successfully.

File: project-example/bin/helper/common.sh [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+#!/bin/bash
+COLOR='\033[1;37m'
+USER=$(whoami)
+
+if [[ ${USER} =~ .*_prod$ ]]; then
+    COLOR="\e[31m"
+fi
+
+LOGO=$(base64 -d <<<"H4sIAExlKF4AA82WORLDIAxFe5+Co1KooHDhKgfMSTJmWL4WZKVJYFQYs0h6fGQfSbb3i6qVavezmvKvdrAeD3Rl2yQAwQfC3jqdkco6zhx4M03tRqJ/Jptaw+HDAjfCo5dB85rmzAsivXBCTU5bWQyh/2eYuW1SYMMIwvpSMTvZnJwwJj4qXcmxNUiGzYJX0jxkXJ5njxijiOLwpG1NBE68J9APuGdjZd7rD6GDKCJYLO4zoALuMEy8i9wcVCCjE2REWlJiNaiPUYkhylxgXyDCxSxBRDRooMCexLSCpPyTrEkephmYBlRGGFbmfGJUQYzHeDE49E2J2zwtISNSlJ1LZ36+zOrsXUCNzCuC3sdmKT9tZ6jkV9VJdC34CQXcirnkD0nAy6oeNnl2v2/s50zcej/0HaJP8u8y2UnsEuzdjg9dwoXwvQsAAA==" | gunzip)
```
Comment (jgimenez) [inline project-example/bin/helper/common.sh:12]: I'd advoacate to remove this. It's not useful and makes the script more complicated.

File: project-example/bin/deployment/build.sh [hunk 5]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+echo "Build Project..."
+
+# Ex. docker-compose -f compose.yml -f compose.{{ENVIRONMENT}}.yml build
```
Comment (jgimenez) [inline project-example/bin/deployment/build.sh:4]: Docker now has the `docker compose` command. Same syntax but embedded in the `docker` binary. This saves you from installing a separate `docker-compose` binary. I'd use that.

Comment (jgimenez) [reply project-example/bin/deployment/build.sh:4]: Also, I'd advocate the runner to not know which environment it is. It's better to create the files compose.yml and compose.override.yml during installation/upgrade. Then, you don't need to know which environment you're in, nor specify the configuration files.

**Pull request merged**